spec/integrations/shop_features_spec.rb

```diff
@@ -7,14 +7,13 @@
 
 RSpec.describe 'Cart features' do
   let(:shop) { Shop.new }
-  let(:product) { Product.new('Szkolenie ruby', 100.20) }
+  let(:product) { Product.new(1, 'Szkolenie ruby', 100.20) }
+  let(:cart) { Cart.new.add_to_cart(product) }
```

Comment:
Tak samo tutaj, jak zepsuje się metoda `add_to_cart` to wszystkie testy ci się posypią